docs/mesos/DockerizedDeployment.md

```diff
@@ -216,5 +226,4 @@ At time of writing Marathon still uses the old Mesos networking, rather than Cal
 [calico]: http://projectcalico.org
```

Comment:
Can you modify the "Test your Cluster" section to use the dockerized test framework instead of telling people to run the python script on their master? Just copy in the stuff you wrote on the new Vagrant guide.